src/app/container/container.component.ts

```diff
@@ -205,7 +205,7 @@ export class ContainerComponent extends Entry {
   }
 
   publishDisable() {
-    return this.refreshMessage !== null || !this.isContainerValid();
+    return (this.refreshMessage !== null && this.refreshMessage !== undefined) || !this.isContainerValid();
```

Comment:
Could simplify to `!this.refreshMessage`, but again, watch out for empty string case if that's relevant.